app-modules/report/src/Filament/Pages/ArtificialIntelligence.php

```diff
@@ -0,0 +1,93 @@
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Report\Filament\Pages;
+
+use Laravel\Pennant\Feature;
+use Filament\Pages\Dashboard;
+use App\Filament\Clusters\ReportLibrary;
+use AdvisingApp\Report\Filament\Widgets\AiStats;
+use AdvisingApp\Report\Filament\Widgets\RefreshWidget;
+use AdvisingApp\Report\Filament\Widgets\PromptsCreatedLineChart;
+use AdvisingApp\Report\Filament\Widgets\SavedConversationsLineChart;
+use AdvisingApp\Report\Filament\Widgets\SpecialActionsDoughnutChart;
+use AdvisingApp\Report\Filament\Widgets\PromptsByCategoryDoughnutChart;
+
+class ArtificialIntelligence extends Dashboard
+{
+    protected static ?string $navigationIcon = 'heroicon-o-document-text';
+
+    protected static ?string $cluster = ReportLibrary::class;
+
+    protected static ?string $navigationGroup = 'Artificial Intelligence';
+
+    protected static ?string $navigationLabel = 'Artificial Intelligence';
+
+    protected static ?string $title = 'Artificial Intelligence';
+
+    protected static string $routePath = 'artificial-intelligence';
+
+    protected static ?int $navigationSort = 10;
+
+    public static function canAccess(): bool
+    {
+        /** @var User $user */
+        $user = auth()->user();
+
+        return $user->can('report-library.view-any') && Feature::active('ai_utilization');
+    }
+
+    public function getWidgets(): array
+    {
+        return [
+            RefreshWidget::make(['pagePrefix' => 'artificial-intelligence']),
+            AiStats::make(['pagePrefix' => 'artificial-intelligence']),
+            SavedConversationsLineChart::make(['pagePrefix' => 'artificial-intelligence']),
+            SpecialActionsDoughnutChart::make(['pagePrefix' => 'artificial-intelligence']),
+            PromptsByCategoryDoughnutChart::make(['pagePrefix' => 'artificial-intelligence']),
+            PromptsCreatedLineChart::make(['pagePrefix' => 'artificial-intelligence']),
```

Comment:
Let's stick the cache tag in a variable above so we don't have to repeat it over and over. And we may want to call it `report-artificial-intelligence` so that it is specific that it is a report.

Reply:
Done